server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -23,33 +20,21 @@
   protected void doGet(HttpServletRequest req, HttpServletResponse res)
       throws ServletException, IOException {
 
-    res.getWriter().write("<h1>Comment Analysis</h1>\n");
-
-    String videoName = "seun omonije";
+    // Video name from the front-end
+    String videoName = "sharks 101";
 
+    // Get response from Youtube API
     String videoIdJson = new YoutubeRequest("search", videoName).getResult();
     String videoId = getVideoId(videoIdJson);
     String commentsJson = new YoutubeRequest("commentThreads", "snippet", videoId).getResult();
 
-    res.getWriter().write(commentsJson);
-
-    res.getWriter().write("<h3>Comments</h3>");
-    String comment = "I love you.";
-    res.getWriter().println(comment);
-
-    res.getWriter().write("<h3>Perspective</h3>");
-    HashMap<String, String> perspectiveMap = analyzeWithPerspective(comment);
-    // Print on the screen (won't make it in final cut)
-    Set set = perspectiveMap.entrySet();
-    Iterator mapIterator = set.iterator();
-    while (mapIterator.hasNext()) {
-      Map.Entry entry = (Map.Entry) mapIterator.next();
-      res.getWriter().println(entry.getKey() + ": " + entry.getValue());
-    }
+    // Convert response into a large string to be passed through for analysis
+    ArrayList<String> commentArray = retrieveComments(commentsJson);
+    String bigCommentString = convertToString(commentArray);
 
-    res.getWriter().write("<h3>NLP</h3>");
-    MagnitudeAndScore nlpObject = analyzeWithNLP(comment);
-    res.getWriter().println("Magnitude: " + nlpObject.magnitude + "Score: " + nlpObject.score);
+    // Analyze with the Perspective and Natural Language APIs
+    HashMap<String, String> perspectiveMap = analyzeWithPerspective(bigCommentString);
+    MagnitudeAndScore nlpObject = analyzeWithNLP(bigCommentString);
```

Comment:
Looks like you currently do nothing with this object. Should you be writing it in your request here?

Reply:
Yes these will be going to the front-end. I plan to do that in the next pr. Should I do it here?

Reply:
Yes. Print them to the front end.

Reply:
Added them to the page